easybuild/easyconfigs/n/nanofilt/nanofilt-2.5.0-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PythonPackage'
+name = 'nanofilt'
+version = '2.5.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/wdecoster/nanofilt'
+description = """Filtering and trimming of long read sequencing data."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/wdecoster/nanofilt/archive/v%(version)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['33c35aad2950145ef66bcf338e68c7c84a20ae62716db24f9c9ccbd881c9a277']
+
+exts_defaultclass = 'PythonPackage'
```

Comment:
this easyconfig has no extensions, please remove this (I suppose it's also why Travis is complaining about `exts_default_options`)